src/assets/css/books.module.scss

```diff
@@ -0,0 +1,275 @@
+@import "./variables.scss";
+
+body {
+  width: 100%;
+  font-family: $RobotoSlab, $Monserrat;
+  color: $black;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+p {
+  font-size: small;
+}
+
+.blue {
+  color: $blue;
+  font-weight: 300;
+  font-size: small;
+  margin-bottom: 5%;
+}
+
+.img {
+  width: auto;
+  height: auto;
+}
+
+.books {
+  width: 100%;
+  background-color: $paleGray;
+
+  &__hr {
+    width: 75rem;
+    height: 0.125rem;
+    margin: 2.5rem 0.063rem 1.813rem 0;
+    border: solid 1px var(--white-three);
+  }
+
+  &__bookCard {
+    // width: 100%;
+    width: 90rem;
+    height: 10.625rem;
+    margin: 2.313rem 1.25rem 0;
+    padding: 2rem 9.188rem 1.625rem 1.688rem;
+    border-radius: 4px;
+    border: solid 1px var(--white-three);
+    background-color: white; /* var(--white-two); */
+    z-index: 1;
+    display: flex;
+    justify-content: space-between;
+
+    h2 a {
+      color: $black;
+      text-decoration: none;
+    }
+
+    &__book {
+      width: 50%;
+      height: 30%;
+      margin: 0.5% 1%;
+
+      &__text {
+        width: 50%;
+
+        &__category {
+          width: 3.063rem;
+          height: 1.125rem;
+          margin: 0 2.438rem 0 0;
+          opacity: 0.5;
+          font-size: 0.875rem;
+          font-weight: bold;
+          font-stretch: normal;
+          font-style: normal;
+          line-height: normal;
+          letter-spacing: normal;
+          color: var(--black-two);
```

Comment:
- [ ] I would kindly suggest you to make you website to be more responsive to fit all screen size please fix this 
- [ ] 

https://github.com/diarisdiakite/bookstore/assets/92458236/2f355bba-3871-4e13-ad4e-49ee7baa21da